server/svix-server/src/v1/endpoints/endpoint/mod.rs

```diff
@@ -203,7 +210,7 @@ impl ModelIn for EndpointIn {
         model.description = Set(description);
         model.rate_limit = Set(rate_limit.map(|x| x.into()));
         model.uid = Set(uid);
-        model.url = Set(url);
+        model.url = Set(url.into());
```

Comment:
You can probably also change the database model to be a Url (or a wrapper of it most likely), and then you don't even need the from/into.

Reply:
I'm not opposed in principle, although in this case I don't think it's worth the effort. `url::Url` doesn't already have the traits for being used as a column value w/sea_orm, so we'd have to add a wrapper type, all the sea_orm traits boilerplate, and still call `.into()` here.

Since we don't interact/mutate the URL outside of `EndpointIn`, I don't think the wrapper type would be much use yet (although it might be worth it in the future).

Reply:
👍🏻